src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Fetch;
+import javax.persistence.criteria.JoinType;
+import javax.persistence.criteria.Root;
+import mate.academy.dao.OrderDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+import mate.academy.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order add(Order order) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(order);
+            transaction.commit();
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException(
+                    "Can't complete order=" + order, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getOrdersHistory(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            CriteriaBuilder criteriaBuilder = session.getCriteriaBuilder();
+            CriteriaQuery<Order> query = criteriaBuilder.createQuery(Order.class);
+            Root<Order> root = query.from(Order.class);
+            Fetch<Object, Object> getTickets = root.fetch("tickets", JoinType.LEFT);
+            Fetch<Object, Object> getMovieSession = getTickets.fetch("movieSession", JoinType.LEFT);
+            getMovieSession.fetch("cinemaHall", JoinType.LEFT);
+            getMovieSession.fetch("movie", JoinType.LEFT);
+            query.where(criteriaBuilder.equal(root.get("user"), user));
+            return session.createQuery(query).getResultList();
```

Comment:
let's rewrite this method using `session.createQuery`
criteria query is harder to read and unnecessary here

Reply:
i like to use it/ i wanted ro rewrite all project to criteriaQuery instead of HQL

Reply:
ok, then please provide a screenshot where it would be visible that we added two tickets to cart 
and then completed order, after that call `getOrdersHistory` and print it to console
screenshot should have that part of code and result in console